keras2onnx/_builtin.py

```diff
@@ -118,43 +118,45 @@ def convert_tf_batch_to_space(scope, operator, container):
                                       starts=[0], ends=[1])
         block_prod = oopb.apply_mul(block_size + block_size,
                                     name=operator.full_name + '_mul_0')
-        padded_block_prod = oopb.apply_pad([block_prod,
-                                            ('const_zero_three', oopb.int64, np.array([0, 3], dtype='int64')),
-                                            ('one', oopb.int64, np.array([1], dtype='int64'))],
-                                           name=operator.full_name + '_pad_0')
+        padded_block_prod = oopb.apply_pad(block_prod,
+                                           name=operator.full_name + '_pad_0',
+                                           pads=[0, 3],
```

Comment:
where does pads come from?

Reply:
This is used for division, it is the dimension of the channel, so 0 means N, 3 means C